scripts/check_frontend_test_coverage.py

```diff
@@ -24,7 +25,17 @@
 
 from core import utils
 
-from typing import List
+from typing import List, Optional
+
+_PARSER = argparse.ArgumentParser(
+    description="""
+Checks the frontend test coverage.
+""")
+
+_PARSER.add_argument(
+    '--files_to_check',
+    type=str
+)
```

Comment:
Please add help text to describe how this argument should be used

Reply:
Done.